### PR TITLE
Add incorrect WCA ID claim count to the DOB form email

### PR DESCRIPTION
**Pull request by jonatanklosko**
Requested by Sébastien. Looks like so:

---

image

### CODE REVIEW

**Comment by SAuroux** [comment]
Looks good to me, thank you. :)

Micro-nit: I would add an empty row before to have some kind of seperation from the user input.

**Comment by jonatanklosko** [comment]
How does it look now?

image

**Comment by SAuroux** [comment]
Good, thanks!

File: WcaOnRails/app/views/mail_form/dob_contact.erb [hunk 1]
```diff
@@ -17,6 +17,13 @@
       <td>Correct birthdate</td>
       <td><strong><%= @resource.dob %></strong></td>
     </tr>
+    <tr style="height: 1em">
+      <td colspan="2"></td>
+    </tr>
+    <tr>
+      <td>Incorrect WCA ID claims</td>
+      <td><strong><%= @resource.incorrect_wca_id_claim_count %></strong></td>
```
Comment (jfly) [inline WcaOnRails/app/views/mail_form/dob_contact.erb]: Maybe put a `/ #{User::MAX_INCORRECT_WCA_ID_CLAIM_COUNT}` in here?

Comment (jonatanklosko) [reply WcaOnRails/app/views/mail_form/dob_contact.erb]: Great idea, done!

**Pull request merged**